src/portal-components/csbc-home-page/AboutPortal.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import { HomePageCardProps } from './HomePageCard'
+import HomePageCardContainer from './HomePageCardContainer'
+import MarkdownSynapse from 'synapse-react-client/dist/containers/MarkdownSynapse'
+
+export type AboutPortalProps = {
+  title: string
+  token?: string
+  cardProps: HomePageCardProps[]
+  logos: string[]
+  ownerId?: string
+  wikiId?: string
+}
+
+const AboutPortal = (props: AboutPortalProps) => {
```

Comment:
I wonder if this will be reused in other portals!  If so, we should refactor into SRC, it's a nice well-written component.